File: exs/mundo_1/java/008.java
/*
 * Desafio 008
 *   Problema: Escreva um programa que leia um valor em metros e o
 *   exiba convertido em centímetros e milímetros.
 *   Resolução do problema:
*/
import java.util.*;

class Main {
    public static void main(String[] args) {
        Scanner x = new Scanner(System.in);
        System.out.print("Metros: ");
        float metros = x.nextFloat();
        System.out.println("Centímetros: " + metros*100 + "cm");
        System.out.println("Milímetros: " + metros*1000 + "mm");
    }
}

File: exs/mundo_1/javascript/008.js
/*
Desafio 008

Problema: Escreva um programa que leia um valor em metros e o exiba convertido em centímetros e milímetros

Resolução do problema:
*/

const input = require('readline-sync').question

let metro = Number(input('Insira um valor em metros: '))

let centimetro = metro*100
let milimetro = metro*1000

console.log(`Metros: ${metro}m`)
console.log(`Centímetros: ${centimetro}cm`)
console.log(`Milímetros: ${milimetro}mm`)